src/libraries/SafeModuleSafeERC20.sol

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: MIT
```

Comment:
So far we used GPL, even if we didn't discuss this too much. I don't necessarily mind keeping MIT but I want to make sure it's intentional. The MIT license allows us to relicense AFAIK.

```suggestion
// SPDX-License-Identifier: GPL-3.0
```